study.md

```diff
@@ -13,6 +13,10 @@ please do the following and be prepared to share and discuss during our next
 class.
 
 Submit detailed answers to these in this file via a pull request.
+You may want to submit pictures for your wireframes and/or user stories.
+[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+has instructions to link to a picture you've uploaded to a service like [Imgur]
+(http://imgur.com/).
```

Comment:
Actually, the answer here is no, since this isn't their project repo, it's the study repo. I don't want them creating a bunch of issues on it. Hopefully they will be able to figure out Imgur/Markdown.